src/frontend/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vite-template-solid",
```

Comment:
Consider changing the name here. Maybe something like `lumen-ui` or something like that.